main/src/main/scala/org/clulab/numeric/actions/NumericActions.scala

```diff
@@ -253,6 +291,14 @@ class NumericActions(seasonNormalizer: SeasonNormalizer, unitNormalizer: UnitNor
 }
 
 object NumericActions {
+  val seasons: Set[String] = Set("spring", "summer", "fall", "autumn", "winter")
+  // Words that typically precede a season that might distinguish it from a similar verb
+  val preSeasons: Set[String] = Set("this", "last", "every")
+  // A common introduction to a season
+  val inThe: Array[String] = Array("in", "the")
+  // Match a 1 to 4 digit year
+  val yearPattern = Pattern.compile("[0-9]{1,4}")
```

Comment:
If this really is for a year, I can only imagine 2 or 4 digits, like summer of '69 or 2023.  Could this be something like "[0-9]{2}|[0-9]{4}"?